third_party/move/move-core/types/src/transaction_argument.rs

```diff
@@ -19,6 +19,7 @@ pub enum TransactionArgument {
     U16(u16),
     U32(u32),
     U256(u256::U256),
+    Serialized(#[serde(with = "serde_bytes")] Vec<u8>),
```

Comment:
Comment above `// END: Added in v6` and maybe comment when why Serialized is added.